Assignment18July/src/AreaOfShapes.java

```diff
@@ -0,0 +1,54 @@
+public class AreaOfShapes {
+	final double pi = 3.14;
+
+	/**
+	 * Find a area of triangle
+	 * 
+	 * @param requires
+	 *            height and width
+	 * @return area of the triangle
+	 */
+	public double areaOfTriangle(double height, double width)
+			throws ArithmeticException {
+		double area = 0.5 * (width * height);
+		return area;
+	}
+
+	/**
+	 * Find a area of circle
+	 * 
+	 * @param requires
+	 *            radius
+	 * @return area of the circle
+	 */
+	public double areaOfCircle(double radius) throws ArithmeticException {
+		double area = pi * radius * radius;
```

Comment:
@meta-namrata-agarwal  Most of the classes have indentation issues. It seems Tab is replace by 8 spaces. change setting in your eclipse editor so that tab should replace 4 spaces